src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -955,11 +917,7 @@ Note that querying for `priority` of 1 and 2 would return no results, because th
 
 #### Paginating queries
 
-[comment]: # (This section was supplemented from - https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/graphqlapi/js/query-data.mdx )
-
-[question]: # (How much do we want to use from this guide which is 2 years old? https://github.com/aws-amplify/docs/blob/main/src/pages/guides/api-graphql/graphql-pagination/q/platform/%5Bplatform%5D.mdx – this content was not added to this page yet.)
-
-To paginate results in AppSync, make a query request specifying a `limit` value. This limits how many results are returned. The response will include a `nextToken` you can use to request the next page of data.
+To paginate results in AWS AppSync, make a query request specifying a `limit` value. This limits how many results are returned. The response will include a `nextToken` you can use to request the next page of data. A `nextToken` is a very long string that represents the cursor to the starting item of the next query made with these filters.
```

Comment:
Note - used Jim's suggestion for rewrite and moved up as the description made more sense before the example code.